doc.go

```diff
@@ -271,25 +271,34 @@ Footnotes:
 
 Note: SQL NULL values are converted to Golang nil values, and vice-versa.
 
+Binding Parameters to Array Variables For Batch Inserts
 
-Binding Time Type
+Version 1.3.9 (and later) of the Go Snowflake Driver supports the ability to bind an array variable to a parameter in an SQL
+INSERT statement. You can use this technique to insert multiple rows in a single batch.
 
-Go's database/sql package limits Go's data types to the following for binding and fetching:
+As an example, the following code inserts rows into a table that contains integer, float, boolean, and string columns. The example
+binds arrays to the parameters in the INSERT statement.
 
-	bool
-	string
-	float64
-	int64
-	time.Time
-	[]byte
+	// Create a table containing an integer, float, boolean, and string column.
+	_, err = db.Exec("create or replace table my_table(c1 int, c2 float, c3 boolean, c4 string)")
+	...
+	// Define the arrays containing the data to insert.
+	intArray := []int{1, 2, 3}
+	fltArray := []float64{0.1, 2.34, 5.678}
+	boolArray := []bool{true, false, true}
+	strArray := []string{"test1", "test2", "test3"}
+	...
+	// Insert the data from the arrays into the table.
+	_, err = db.Exec("insert into my_table values (?, ?, ?, ?)", intArray, fltArray, boolArray, strArray)
+
+Note: For alternative ways to load data into the Snowflake database (including bulk loading using the COPY command), see
+Loading Data Into Snowflake (https://docs.snowflake.com/en/user-guide-data-load.html).
 
-Fetching data isn't an issue because the database data type is provided along
-with the data so the Go Snowflake Driver can translate Snowflake data types to
-Go native data types.
```

Comment:
Most of this didn't seem to have much to do with binding a time variable to a parameter, so I removed it. There's also an earlier section that discusses the types supported by the driver, so I didn't think there should be another list here.